src/vivarium/framework/artifact/artifact.py

```diff
@@ -214,15 +214,15 @@ def __str__(self):
         return out
 
 
-def _to_tree(keys: List[str]) -> Dict[str, Dict[str, List[str]]]:
-    out = defaultdict(lambda: defaultdict(list))
+def _to_tree(keys: list[str]) -> dict[str, dict[str, list[str]]]:
+    out: defaultdict[str, dict[str, list[str]]] = defaultdict(lambda: defaultdict(list))
     for k in keys:
         key = k.split(".")
         if len(key) == 3:
             out[key[0]][key[1]].append(key[2])
         else:
             out[key[0]][key[1]] = []
-    return out
+    return dict(out)
```

Comment:
why did you add this?

Reply:
It wanted a dict returned and it originally gets defined as a default dict.